tests/cases/conformance/async/es2017/asyncImportedPromise_es2017.ts

```diff
@@ -0,0 +1,10 @@
+// @target: es2017

```

Comment:
This test is unnecessary as its only purpose is to verify the checker with respect to async functions, which is already handled by existing tests.